### PR TITLE
Fixes #14872: Deduplicate cfengine binaries

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/14872

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

File: rudder-agent/SOURCES/rudder-agent-postinst [hunk 1]
```diff
@@ -84,10 +84,6 @@ if type slibclean > /dev/null 2>&1; then
   slibclean
 fi
 
-# Copy CFEngine binaries (Changed location from sbin/ to bin/ in version 3.4)
-cp -r -f /opt/rudder/bin/cf-* ${CFE_DIR}/bin/
```
Comment (amousset) [inline rudder-agent/SOURCES/rudder-agent-postinst]: this is also done in the system techniques

File: rudder-server-root/SPECS/rudder-server-root.spec [hunk 2]
```diff
@@ -83,25 +83,6 @@ LOG_FILE="/var/log/rudder/install/%{name}.log"
...
-  mkdir -p /var/rudder/tmp
-
-  if [ -f /etc/init.d/rudder-server ]
-  then
-    # If old rudder service is here and enabled
-    if type chkconfig > /dev/null
-    then 
-      if chkconfig --list rudder-server 2>&1 | grep -q -e 3:on -e B:on
-      then
-        touch /var/rudder/tmp/migration-rudder-service-enabled-server
```
Comment (amousset) [inline rudder-server-root/SPECS/rudder-server-root.spec]: This is a 5.0 -> 5.1 migration

File: rudder-agent/SOURCES/rudder-agent-postinst (hunk 1 above)
Comment (amousset) [reply rudder-agent/SOURCES/rudder-agent-postinst]: https://github.com/Normation/rudder-techniques/pull/1442

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Pull request merged**